app/views/thesis/new.html.erb

```diff
@@ -7,101 +7,106 @@
+                        required: true,
+                        validate: { presence: true },
+                        label: 'Thesis title',
+                        label_html: { class: 'col1q' },
+                        wrapper_html: { class: 'field-wrap layout-1q3q layout-band' },
+                        input_html: { class: 'field field-text col3q',
+                          data: { msg: Thesis::VALIDATION_MSGS[:title] }
+                        } %>
 
+    <%= f.simple_fields_for :users do |u| %>
+      <% if u.object.id == current_user.id %>
+        <%= u.input :preferred_name, label: 'Name as it appears on thesis',
+                                     label_html: { class: 'col1q' },
+                                     wrapper_html: { class: 'field-wrap layout-1q3q layout-band' },
+                                     input_html: { class: 'field field-text col3q' },
+                                     hint: '(Enter as: Last/Surname, First/Given name Middle name)',
+                                     hint_html: { class: 'col3q' } %>
+      <% end # If IDs match %>
+    <% end # simple_fields_for loop %>
+
+    <%= f.input :coauthors, label: 'Co-author (if applicable)',
+                            label_html: { class: 'col1q' },
+                            wrapper_html: { class: 'field-wrap layout-1q3q layout-band' },
+                            input_html: { class: 'field field-text col3q' },
+                            hint: '(Enter as: Last/Surname, First/Given name Middle name)',
+                            hint_html: { class: 'col3q' } %>
+
+    <% # TODO: Make Departments an "add-another" field %>
     <%= f.association :departments, label_method: :name_dw, as: :select,
                          collection: Department.order(:name_dw),
                          validate: { presence: true },
                          include_hidden: false,
-                         wrapper_html: { class: 'field-wrap select' },
-                         input_html: { class: 'field field-select', multiple: false,
+                         label_html: { class: 'col1q' },
+                         wrapper_html: { class: 'field-wrap select layout-band layout-1q3q' },
+                         input_html: { class: 'field field-select col3q', multiple: false,
                            data: { msg: Thesis::VALIDATION_MSGS[:departments] },
                            'aria-describedby': 'thesis_departments_ids-hint'
                          },
-                         hint: 'Select your primary department as listed on your title page',
-                         hint_html: { id: 'thesis_departments_ids-hint' } %>
+                         hint: 'Select your primary department as listed on your title page. If additional departments are needed, use the Notes field below.',
+                         hint_html: { id: 'thesis_departments_ids-hint', class: 'col3q' } %>
 
+    <% # TODO: Make Degrees an "add-another" field %>
```

Comment:
I've fine with this being later work, but when this lands lets make sure we have a ticket to follow up on this.

Reply:
I've created https://mitlibraries.atlassian.net/browse/ETD-223 as a related ticket.